pom.xml

```diff
@@ -52,15 +52,15 @@
 
     <!-- This allows you to test feature branches with jitpack -->
     <armeria.groupId>com.linecorp.armeria</armeria.groupId>
-    <armeria.version>1.17.2</armeria.version>
+    <armeria.version>1.26.3</armeria.version>
```

Comment:
ITElasticsearchClientInitialization seems to not behave nicely. I know a lot has changed between these two here. @openzipkin/armeria does anyone have time to diagnose?

https://github.com/openzipkin/zipkin/actions/runs/7099979222/job/19325100225?pr=3614

Reply:
cc also @ikhoon who isn't in that list but maybe should be as has helped in the past!